components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -134,6 +134,17 @@ use the Habitat Builder UI to create the necessary origins before running the
 ### Operating Habitat Builder
 Habitat Builder uses the same mechanisms that Chef Automate does for [backups]({{<relref "backup.md">}}), [log management]({{<relref "log-management.md">}}), and [uninstallation]({{<relref "troubleshooting.md#uninstalling-chef-automate">}}).
 
+To change the log level for Habitat Builder only, create a TOML file that contains the partial configuration below. Uncomment and change settings as needed, then run `chef-automate config patch </path/to/your-file.toml>` to deploy your change.
+
+```toml
+[builder_api.v1.sys.log]
+level = "debug"
+scoped_levels = ["tokio_core=error"]
+```
+
+Set the log level to `error` for normal operation and to `debug` for any debugging. Due to
```

Comment:
this is not correct. `level` is whatever automate accepts (`debug`, `info`, `warning`, `error`, `fatal`, `panic` I think). I would avoid mentioning scoped_levels entirely as its an implementation detail that we can set with support when needed

Reply:
@jaym I've removed `scoped_levels` from the config example and the incorrect text about levels.